python/ql/test/library-tests/frameworks/internal-ql-helpers/PoorMansFunctionResolutionTest.ql

```diff
@@ -11,6 +11,10 @@ module InlinePoorMansFunctionResolutionTest implements TestSig {
     exists(Function func, DataFlow::Node ref |
       ref = poorMansFunctionTracker(func) and
       not ref.asExpr() instanceof FunctionExpr and
+      // exclude the name of a defined function
+      not exists(FunctionDef def | def.getDefinedFunction() = func |
+        ref.asExpr() = def.getATarget()
+      ) and
       // exclude things like `GSSA variable func`
       exists(ref.asExpr()) and
```

Comment:
Presumably not needed now? (Or at the very least, the comment should be updated.)